api/src/main/java/org/openmrs/module/sync/SyncUtilTransmission.java

```diff
@@ -603,4 +565,60 @@ public static SyncTransmissionResponse doFullSynchronize(ReceivingSize size, Int
 			return response;
 		}
 	}
+
+	private static SyncTransmissionResponse transmitAndProcessResponse(RemoteServer server, IItem toTransmit) {

```

Comment:
Can you please split this method dividing responsibilities to be more clear and also be easy to test.

Reply:
No, sorry I can't this is not my code. We don't want to change stuff that is working.